src/OSFramework/Maps/Event/Shape/ShapeProviderEvent.ts

```diff
@@ -14,17 +14,25 @@ namespace OSFramework.Maps.Event.Shape {
          * @param mapId Id of the Map that is raising the event
          * @param shapeId Id of the Shape that is raising the event
          * @param eventName Name of the event that got raised
+         * @param args Event properties as coordinates and location
          */
         public trigger(
             // eslint-disable-next-line @typescript-eslint/no-unused-vars
             mapId: string,
             shapeId: string,
-            eventName: string
+            eventName: string,
+            args: any
```

Comment:
Let's avoid using `any`. Use instead `unknown`.

Reply:
I've checked and "args" will always be an object containing the coordinates, so I changed its name to "coords", created the new type OSShapeCoordinates and assigned to the argument.